### DISCUSSION TITLE
Is SSG (next export) without a server running supported for new Nextjs 13 app directory?

### DISCUSSION THREAD

**Discussion by CaptShmo** [vercel/next.js#42602] [Help] [answered]
Hey all, I am currently working on a project where a lot of our sites are going to be flat html/css/js where we want to host our sites in s3 buckets or similar system that doesn't require a server running 24/7 to handle requests. I am trying to do a next export right now with a brand new Nextjs13 install with just a single page.js in the app folder and it is telling me it can't find a module for page "/". Is my use case currently supported or planned for nextjs13? My goal is to use the new nested layout functionality supported by the app dir but still statically render the site as flat html etc. Thanks!

**Comment by arielvieira** [comment, marked as answer]
@CaptShmo Right now Next.js doesn't support `next export` in `app` directory, only in `pages` directory.

According to the roadmap they are planning to add this in the future, https://beta.nextjs.org/docs/app-directory-roadmap#planned-features.

If you need to use next export you should use the pages directory instead.

**Reply by CaptShmo** [reply]
Okay, I saw this page during my search but somehow missed that line; thank you!
